draco/synthesis/mockcatalog.py

```diff
@@ -0,0 +1,1045 @@
+"""Tasks for making mock catalogs.
+
+See Usage section for usage.
+
+Pipeline tasks
+==============
+
+.. autosummary::
+    :toctree:
+
+    SelFuncEstimator
+    ResizeSelFuncMap
+    PdfGeneratorBase
+    PdfGeneratorUncorrelated
+    PdfGeneratorNoSelfunc
+    PdfGeneratorWithSelfunc
+    MockCatGenerator
```

Comment:
For some kind of parallelism with the module name, maybe this task should be called `MockCatalogGenerator`